manager/Dockerfile

```diff
@@ -1,9 +1,10 @@
 # Copyright 2020 IBM Corp.
 # SPDX-License-Identifier: Apache-2.0
 
-FROM gcr.io/distroless/static:nonroot
-WORKDIR /
-COPY manager .
-USER nonroot:nonroot
+FROM ubi8/ubi-minimal
+ENV HOME=/tmp
+WORKDIR /tmp
```

Comment:
Just wondering, is it a best practice to use /tmp as the workdir?

Reply:
This is something we've taken to doing for openshift.  Often, you'll be running with an SCC that randomizes the user ID you run as, so you want to be sure you've got access to execute the binary, and access to write any files to $HOME or PWD if the application requires it.